scripts/tests/coordinate_transform_benchmarks.py

```diff
@@ -147,6 +175,15 @@ def oldvsnew_diagram():
     plt.xlabel('Error (arcsec)')
     plt.ylabel('Counts')
 
+    # Make histogram of differences using the absolute distance in arcsec-this graph has no wrapping issues
+    plt.figure(8)
+    nieuw = np.array([angle_between_horizontal(htoe[i][0], htoe[i][1], htoea[i][0], htoea[i][1]) for i in range(len(htoe))])
+    nieuw = nieuw / 2 / np.pi * 360 * 3600  # Take difference and convert to arcsec
```

Comment:
This might be a bit unclear to people not used to the [operator precedence](https://docs.python.org/3.6/reference/expressions.html#operator-precedence) in Python, because here `/` and `*` are equal. Please clarify it by using brackets, e.g.:
``` python
nieuw = nieuw * 360 * 3600 / (2 * np.pi)
```

Or even
``` python
nieuw *= 360 * 3600 / (2 * np.pi)
```